### DISCUSSION TITLE
add invisible stall to game

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17727] [Feature requests] [answered]
similar to parkitect's custom food stands, add an invisible stall to game (similar to invisible pathing) that has a menu that lets you decide what it sells out of every shop item in the game

**Comment by ocalhoun6** [comment]
You can already build whatever stall you want, selling whatever that stall sells, and then make it invisible using the tile inspector. The only difference is that you have to decide what you want the stall to sell _before_ you build it.

It would also be possible to implement invisible stalls as custom objects, though you'd need a different object for each different combination of things you want the stall to sell.

**Comment by karst** [comment, marked as answer]
There's already custom objects for this. My favourite isn't an invisible one, but just a cashier and a counter version of it. Idm seeing that in the game, but I think it works fine as a custom object rather than official object.
